src/Workspaces/Remote/ServiceHub/Host/SolutionAssetCache.cs

```diff
@@ -34,8 +40,7 @@ internal sealed class SolutionAssetCache
         /// </summary>
         private readonly TimeSpan _gcAfterTimeSpan;
 
-        private readonly ConcurrentDictionary<Checksum, Entry> _assets =
-            new ConcurrentDictionary<Checksum, Entry>(concurrencyLevel: 4, capacity: 10);
+        private readonly ConcurrentDictionary<Checksum, Entry> _assets = new();
```

Comment:
deferring to the runtime to determine the right defaults, absent any documentation that these defaults are better.

Reply:
@CyrusNajmabadi Did you think there was any specific concern with the defaults? Would you want to change the defaults in a separate PR that's merged a few days later so if this does impact things we might understand what changed things?

Reply:
i can restore this.  my recollection is that we did these sorts of capacities originally inside VS because ConcurrentDict exhibited poor memory performance there.  But i think that's an aspect of netstandard2.  Netcore6 and above are fine. 

will restore though as your point is valid.